src/main/java/ru/skypro/homework/service/CommentService.java

```diff
@@ -0,0 +1,20 @@
+package ru.skypro.homework.service;
+
+import ru.skypro.homework.dto.Comment;
+import ru.skypro.homework.dto.Comments;
+
+public interface CommentService {
+    default Comments getCommentsByAdId(Long adId) {
```

Comment:
Методы не дефолтные, их нужно будет реализовать 
```suggestion
    Comments getCommentsByAdId(Long adId);
```